ast/expressions_test.go

```diff
@@ -301,3 +301,15 @@ func (tc *testExpressionsSuite) TestMaxValueExprRestore(c *C) {
 	}
 	RunNodeRestoreTest(c, testCases, "alter table posts add partition ( partition p1 values less than %s)", extractNodeFunc)
 }
+
+func (tc *testExpressionsSuite) TestExistsSubqueryExprRestore(c *C) {
+	testCases := []NodeRestoreTestCase{
+		// TODO: Test for subquery when it's implemented
+		//{"EXISTS (select 2)", "EXISTS (select 2)"},
```

Comment:
i think we should merge this after `SelectStmt` implemented.

Reply:
OK